content/en/logs/workspace/_index.md

```diff
@@ -0,0 +1,215 @@
+
+### Transformation cell
+
+Click the **transformation** tile to add a cell for filtering, aggregating, and extracting data.
+
+1. Click the **Transformation** tile.
+1.  Select the data source you want to transform  in the **Source dataset** drop down menu.
+1. Click **+** to add a **Filter**, **Parse**, or **Aggregate** function.
+    - For **Filter**, add a filter query for the dataset.
+    - For **Parse**, enter grok syntax to extract data into a separate column. In the **from** drop down menu, select the column the data is getting extracted from. See the [column extraction example](#column-extraction-example).
+    - For **Aggregate**, select what you want to group the data by in the drop down menus.
+1. Click the download icon to export the dataset into a CSV.
+
+#### Column extraction example
+
+If you want to extract the customer ID from the message to a separate column in this example dataset:
+
+| timestamp           | host             | message                            |
+| ------------------- | ---------------- | ---------------------------------- |
+| May 29 11:09:28.000 | shopist.internal | Submitted order for customer 21392 |
+| May 29 10:59:29.000 | shopist.internal | Submitted order for customer 38554 |
+| May 29 10:58:54.000 | shopist.internal | Submitted order for customer 32200 |
+
+Use the following grok syntax to extract the customer ID from the message column and add it to a new column called `customer_id`.
+
+```
+Submitted order for customer %{notSpace:customer_id}`
+```
+
+This is the dataset in the transformation cell after the extraction:
+
+| timestamp           | host             | message                            | customer_id |
+| ------------------- | ---------------- | ---------------------------------- | ----------- |
+| May 29 11:09:28.000 | shopist.internal | Submitted order for customer 21392 | 21392       |
+| May 29 10:59:29.000 | shopist.internal | Submitted order for customer 38554 | 38554       |
+| May 29 10:58:54.000 | shopist.internal | Submitted order for customer 32200 | 32200       |
+
+### Text cell
+
+Click the **Text** cell to add a markdown cell so you can add information and notes.
+
+## An example workspace
+
+{{< img src="logs/workspace/workspace_datasets_example.png" alt="The workspace datasets" style="width:100%;" >}}
+
+This example workspace has:
+-  Three data sources:
+	- `transaction_start_logs`
+	- `transaction_execution_logs`
+	- `trading_platform_users`
+- Three derived datasets, which are the results of data that has been transformed from filtering, grouping, or querying using SQL:
+    - `parsed_executed_logs`
+    - `transaction_record`
+    - `transaction_record_with_names`
+
+- One treemap visualization.
+
+This diagram shows the different transformation and analysis cells the data sources go through.
+
+{{< img src="logs/workspace/flowchart.png" alt="A flowchart showing the steps that the data sources go through" style="width:60%;"  >}}
```

Comment:
Will open up a brand design card for this but it probably won't be done until after DASH.

Reply:
Done.